torch/_tensor_str.py

```diff
@@ -266,6 +266,23 @@ def _str(self):
         if values.numel() == 0:
             values_str += ', size=' + str(tuple(values.shape))
         tensor_str = indices_prefix + indices_str + '),\n' + ' ' * indent + values_prefix + values_str + ')'
+    elif self.is_quantized:
+        suffixes.append('size=' + str(tuple(self.shape)))
+        if not has_default_dtype:
+            suffixes.append('dtype=' + str(self.dtype))
+        is_quantized_prefix = 'is_quantized=True'
+        qscheme_prefix = 'qscheme=per_tensor_affine'
+        values_prefix = 'float_values=tensor('
+        values = self.dequantize()
+        values_str = _tensor_str(values, indent + len(values_prefix))
+        if values.numel() == 0:
+            values_str += ', size=' + str(tuple(values.shape))
+        int_repr_prefix = 'int_repr=tensor('
+        int_repr = self.int_repr()
+        int_repr_str = _tensor_str(int_repr, indent + len(int_repr_prefix))
+        if int_repr.numel() == 0:
+            int_repr_str += ', size=' + str(tuple(int_repr.shape))
```

Comment:
why can't you handle the size things automatically through the recursive calls?

Reply:
oh this is copied from sparse, we don't need this for quantized.